crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -15,7 +15,7 @@ use uuid::Uuid;
 /// Emitted when [`crate::SceneInstance`] becomes ready to use.
 ///
 /// See also [`SceneSpawner::instance_is_ready`].
-#[derive(Event)]
+#[derive(Debug, Event, PartialEq)]
```

Comment:
I added these for the unit test, but I can figure it out without, so let me know if you want me to remove them.

Reply:
Obsolete after #11003 was merged.